webapp/src/components/gitlab_project_selector/index.tsx

```diff
@@ -34,16 +34,25 @@ const GitlabProjectSelector = ({theme, required, onChange, value, addValidate, r
 
     const loadProjects = async () => {
         setIsLoading(true);
-        await dispatch(getProjects());
+        const res: any = await dispatch(getProjects());
```

Comment:
We want to avoid `any` when possible

```suggestion
        const res = (await dispatch(getProjects())) as {error?: ErrorType};
```